packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -584,6 +590,7 @@ class _UiKitViewState extends State<UiKitView> {
     }
     _initialized = true;
     _createNewUiKitView();
+    _focusNode = FocusNode(debugLabel: "UiKitView(id: $_id)");
```

Comment:
You wouldn't need to set this if you declared it as `  late final FocusNode _focusNode = FocusNode(debugLabel: 'UiKitView(id: $_id)');`
```suggestion
```
But really, is this the right place to set this?  What if the `_id` changes?  It seems like it should be initialized in `setState`?
Also this should be single-quotes.